galaxy/importer/loaders/role.py

```diff
@@ -81,6 +81,8 @@ def _get_dependencies(self, metadata):
         return dependencies
 
     def _validate_tag(self, tag):
+        if not isinstance(tag, str):
+            return False
```

Comment:
Looks goods to me.

I was curious how/if the collection import models and validators handle it (galaxy.importer.models.CollectionInfo) and it looks like the validators there already handle it.

Code as is in this PR looks good to me.